.github/workflows/docker.yaml

```
@@ -26,8 +29,12 @@ jobs:
       - name: Install Dev Requirements
         run: pip install -r dev-requirements.txt
 
-      - name: Build Fidesctl
+      - name: Build Fidesctl Prod
         run: nox -s "build(prod)"
```

Should this only run on tags since it is prod?

```suggestion
      - name: Build Fidesctl Prod
        if: ${{ env.TAG }}
        run: nox -s "build(prod)"
```

---

no, whether we're pushing a `dev` or a `prod` image, we want to use the prod build stage. For example, `build(dev)` doesn't actually build and install the image, since its meant for local development. `build(prod)` is what we want any time we are pushing the image to dockerhub

---

I'll update the comments to make this more accurate/helpful